generators/chipyard/src/main/scala/DigitalTop.scala

```diff
@@ -32,6 +32,7 @@ class DigitalTop(implicit p: Parameters) extends ChipyardSystem
   with chipyard.example.CanHavePeripheryStreamingPassthrough // Enables optionally adding the DSPTools streaming-passthrough example widget
   with nvidia.blocks.dla.CanHavePeripheryNVDLA // Enables optionally having an NVDLA
   with chipyard.clocking.HasChipyardPRCI // Use Chipyard reset/clock distribution
+  with FFTGenerator.HasPeripheryFFT
```

Comment:
This should be `fftgenerator`. Also the name of the trait should be `CanHavePeripheryFFT`

Reply:
+1 (I assume you have changed the trait to "enable/disable" the FFT)